### ISSUE TITLE
ScrollablePane&DetailsList: Elements that have scrollable content must be accessible by keyboard

### ISSUE THREAD

**Issue by kidokidozh** [microsoft/fluentui#18232]
### Describe the issue:

Accessibility Insights tool fails when using `ScrollablePane` together with `DetailsList`

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/kidokidozh/pen/vYxXpGd

![image](https://user-images.githubusercontent.com/58833558/118593197-dbe5f880-b7d9-11eb-9309-d3853f998cfe.png)

**Comment by ecraig12345** [comment]
ScrollablePane's current implementation is very bad for accessibility when paired with complex interactive content like a DetailsList, so we don't recommend using them together (there's actually an open PR #18249 removing that example from our website). 

If you tell us more about what you're trying to do (in general, no product details please) maybe we can suggest an alternative.

**Comment by kidokidozh** [comment]
@ecraig12345 We are using `ScrollablePane` and `DetailsList` together to have a sticky header table. Do you have any suggestion on the best practices?

**Comment by ecraig12345** [comment]
@kidokidozh The second example on this page shows one possible approach. https://fabricweb.z5.web.core.windows.net/oufr/8.21.1/#/examples/scrollablepane
